packages/graphql-mini-transforms/src/document.ts

```diff
@@ -66,7 +68,24 @@ export function extractImports(rawSource: string) {
   return {imports: [...imports], source};
 }
 
-function removeUnusedDefinitions(document: DocumentNode) {
+export function toSimpleDocument<Data, Variables, DeepPartial>(
+  document: DocumentNode<Data, Variables, DeepPartial>,
+): SimpleDocument<Data, Variables, DeepPartial> {
```

Comment:
I guess this quietly changes the type for consumers not using simple as well? We should make sure to call that out in CHANGELOG as well

Reply:
I think it is only conditionally exported in this shape (see `jest`/ `webpack` sibling files, which consume these utilities)